plugin/vcs/gitlab/gitlab.go

```diff
@@ -58,12 +58,11 @@ type WebhookPost struct {
 	// Saying that, delivering a souding dry run solution would be great and hopefully we can achieve that one day.
 	// MergeRequestsEvents  bool   `json:"merge_requests_events"`
 	PushEventsBranchFilter string `json:"push_events_branch_filter"`
-	// TODO(tianzhou): This is set to false, be lax to not enable_ssl_verification
-	EnableSSLVerification bool `json:"enable_ssl_verification"`
+	EnableSSLVerification  bool   `json:"enable_ssl_verification"`
```

Comment:
I think it makes more sense to document this TODO on the call sites (thus moved it up), unless we are doing something to enforce the behavior here (i.e. hijacking JSON encoding to make `"enable_ssl_verification"` to be always `false`).